### PR TITLE
Update current scripts used for setting up s390x runners

### PR DESCRIPTION
**Pull request by AlekseiNikiforovIBM**
Update current scripts used for setting up s390x runners

Just a documentation update.

### CODE REVIEW

**Review by malfet — APPROVED** [review]
Looks fine, though some changes seems unrelated

File: .github/scripts/s390x-ci/self-hosted-builder/fs/usr/bin/entrypoint [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```
Comment (malfet) [inline .github/scripts/s390x-ci/self-hosted-builder/fs/usr/bin/entrypoint:1]: Why this change is needed?

File: .github/scripts/s390x-ci/self-hosted-builder/helpers/app_token.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```
Comment (malfet) [inline .github/scripts/s390x-ci/self-hosted-builder/helpers/app_token.sh:1]: On some OSes bash is in `/usr/bin/` rather than in `/bin`, this is why it's often prefixed with `/usr/bin/`

File: .github/scripts/s390x-ci/self-hosted-builder/fs/usr/bin/entrypoint (hunk 1 above)
Comment (AlekseiNikiforovIBM) [reply .github/scripts/s390x-ci/self-hosted-builder/fs/usr/bin/entrypoint:1]: You're correct. Those changes are not needed. I'll update it everywhere to previous version.

**Comment by AlekseiNikiforovIBM** [comment]
@malfet , could you please take another look here?

**Review by huydhn — APPROVED** [review]
It looks like we can reuse the helpers function in other cases (as long as they have an app similar to s390x-pytorch-runner installed)
